modules/ROOT/pages/revisionhistory.adoc

```diff
@@ -0,0 +1,165 @@
+* Current Version: is added on opening the {pluginname} with the editor's current content. If included, it will become the `latest version` and placed at the top of the revisions list.
+
+[TIP]
+If both versions are the same, only the `current version` is kept.
+
+For documents with saved versions, the `initial version` is disregarded by the {pluginname} assuming it's already part of the version history. The current version is only discarded if it matches the closest saved version.
+
+The below table summaries how the initial and current versions are handled in the {pluginname}:
+
+[cols="1,1,1,3", options="header"]
+|===
+|Initial Version |Different Current Version |Has Saved Versions |Expect
+|No |No |No |No versions are shown
+|Yes |No |No |Only the current version is shown
+|Yes |Yes |No |Both the initial and current versions are shown
+|Yes |Yes |Yes |The saved versions and the current version are shown
+|Yes |No |Yes |Only the saved versions are shown
+|No |Yes |Yes |The saved versions and the current version are shown
+|No |No |Yes |Only the saved versions are shown
+|===
+
+[NOTE]
+`Different current version` implies whether the current version is different than the initial version. When the editor is loaded with content, the two versions are the same.
+
+== Limitations
+
+1. There are several known limitations when it comes to the plugin's content diffing capability.
+
+* For the listed content below, the {pluginname} might not be able to render them as how they are rendered in the editor and the diffing might not correctly reflects the changes on them:
+** Annotated content or content whose visual representation is provided by a plugin, such as comments or page breaks.
+** Custom content if its visual representation can be adjusted using CSS, integrators can address this by providing styling via the xref:revisionhistory_css_url[`+revisionhistory_css_url+`] option.
+* Code within code blocks may not be accurately differentiated, excluding changes in newlines.
+
+2. There is no integration for saving a version yet. For now, a workaround is to add a custom toolbar button. For example:
+
+[source,js]
+----
+const revisions = [];
+
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'revisionhistory',
+  toolbar: 'revisionhistory saveversion',
+  setup: (editor) => {
+    // Save the current content of the editor as a new version
+    editor.ui.registry.addButton('saveversion', {
+      text: 'Save a version',
+      onAction: () => {
+        // Change this according to your setup
+        const revision = {
+          content: editor.getContent({ format: 'raw' }),
+          revisionId: (revisions.length + 1).toString(),
+          createdAt: new Date().toISOString()
+        };
+        revisions.unshift(revision);
+      }
+    });
+  },
+  revisionhistory_fetch: () => Promise.resolve(revisions)
+});
+----
```

Comment:
Q: are we moving this for `initial release`.
```suggestion
== Limitations

1. There are several known limitations when it comes to the plugin's content diffing capability.

* For the listed content below, the {pluginname} might not be able to render them as how they are rendered in the editor and the diffing might not correctly reflects the changes on them:
** Annotated content or content whose visual representation is provided by a plugin, such as comments or page breaks.
** Custom content if its visual representation can be adjusted using CSS, integrators can address this by providing styling via the xref:revisionhistory_css_url[`+revisionhistory_css_url+`] option.
* Code within code blocks may not be accurately differentiated, excluding changes in newlines.

2. There is no integration for saving a version yet. For now, a workaround is to add a custom toolbar button. For example:

[source,js]
----
const revisions = [];

tinymce.init({
  selector: 'textarea',  // change this value according to your HTML
  plugins: 'revisionhistory',
  toolbar: 'revisionhistory saveversion',
  setup: (editor) => {
    // Save the current content of the editor as a new version
    editor.ui.registry.addButton('saveversion', {
      text: 'Save a version',
      onAction: () => {
        // Change this according to your setup
        const revision = {
          content: editor.getContent({ format: 'raw' }),
          revisionId: (revisions.length + 1).toString(),
          createdAt: new Date().toISOString()
        };
        revisions.unshift(revision);
      }
    });
  },
  revisionhistory_fetch: () => Promise.resolve(revisions)
});
----
```

Reply:
It is already moved